src/utils/dom-tranform-stream.js

```diff
@@ -143,7 +144,7 @@ export class DomTransformStream {
       this.targetBodyPromise_,
       this.headPromise_,
     ]).then((resolvedElements) =>
-      this.vsync_.mutatePromise(() => {
+      this.transferThrottle_(() => {
```

Comment:
You probably have to do here:

```
const transferThrottle = this.transferThrottle_;
return transferThrottle(...);
```

This because calling it as `this.transferThrottle_` will push `this` to its caller stack, which is not correct - this function is not a method on this class.

Reply:
done.